web/vtadmin/README.md

```diff
@@ -2,7 +2,7 @@
 
 ## Prerequisites
 
-- [node](https://nodejs.org) >= 16.13.0 LTS
+- [node](https://nodejs.org) >= 16.19.0 LTS
```

Comment:
is this required?

Reply:
@ajm188 node 16.19.0 is required to run vtadmin yes: https://github.com/vitessio/vitess/blob/main/web/vtadmin/package.json#L6